test_scripts/RC/InteriorVehicleData_cache/033_GetInteriorVD_subscribe_false_by_app_unregistration_2_modules.lua

```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/InteriorVehicleData_cache/common_interiorVDcache')
+local commonRC = require('test_scripts/RC/commonRC')
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local Function ]]
+local function unregistrationApp()
+  local rpc = "GetInteriorVehicleData"
+  EXPECT_HMICALL(commonRC.getHMIEventName(rpc))
+  :Do(function(_, data)
+      if data.params.moduleType == "CLIMATE" then
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+        common.getHMIResponseParams(rpc, "CLIMATE", false))
+      else
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+        common.getHMIResponseParams(rpc, "RADIO", false))
+      end
+    end)
+  :ValidIf(function(_, data)
+      local isError = false
+      local ExpectedResult
+      if data.params.moduleType == "CLIMATE" then
+        ExpectedResult = common.getHMIRequestParams(rpc, "CLIMATE", 1, false)
+      else
+        ExpectedResult = common.getHMIRequestParams(rpc, "RADIO", 1, false)
+      end
+      isError = commonFunctions:is_table_equal(data.params, ExpectedResult)
+      if isError == false then
+        return isError, "Parameters in RC.GetInteriorVehicleData are not match to expected result.\n" ..
+          "Actual result:" .. common.tableToString(data.params) .. "\n" ..
+          "Expected result:" ..common.tableToString(ExpectedResult) .."\n"
+      end
+      return true
+    end)
+  :Times(2)
+  local mobSession = common.getMobileSession(1)
+  local hmiAppId = commonRC.getHMIAppId(1)
+  local cid = mobSession:SendRPC("UnregisterAppInterface",{})
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", { appID = hmiAppId, unexpectedDisconnect = false })
+  mobSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Update preloaded file for RC permissions", common.updatePreloadedPT)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register app", common.registerAppWOPTU, { 1 })
+runner.Step("Activate app", common.activateApp, { 1 })
+
+runner.Title("Test")
+
+runner.Step("GetInteriorVehicleData with subscribe=true " .. common.modules[1], common.GetInteriorVehicleData,
+  { common.modules[1], true, true, 1 })
+runner.Step("GetInteriorVehicleData with subscribe=true " .. common.modules[2], common.GetInteriorVehicleData,
+  { common.modules[2], true, true, 1 })
+runner.Step("RC.GetInteriorVehicleData with subscribe=false by app unregistration " .. common.modules[1],
```

Comment:
@GetmanetsIrina `common.modules[1]` is redundant since unsubscription will happen for both modules